stress/shell/src/main/java/alluxio/stress/cli/client/StressClientIOBench.java

```diff
@@ -117,7 +117,7 @@ public void prepare() throws Exception {
       // set hdfs conf for preparation client
       Configuration hdfsConf = new Configuration();
       hdfsConf.set(PropertyKey.Name.USER_FILE_DELETE_UNCHECKED, "true");
-      hdfsConf.set(PropertyKey.Name.USER_FILE_WRITE_TYPE_DEFAULT, "MUST_CACHE");
+      hdfsConf.set(PropertyKey.Name.USER_FILE_WRITE_TYPE_DEFAULT, mParameters.mWriteType);
```

Comment:
doesn't this change the write type of the test from MUST_CACHE to whichever the alluxio default is, which i think i' found as ASYNC_THROUGH in PropertyKey.java? in order to preserve the same test results for an existing test, it would need to explicitly configure the write type to be MUST_CACHE to get the same test results?

Reply:
Here the writeType is only used for the preparation client. Also this is the preparation step, so even if the behavior changes (by incurring extra work), the performance test result should not be affected.